ovos_utils/configuration.py

```diff
@@ -146,31 +240,13 @@ def store(self, path=None):
 
 class MycroftUserConfig(LocalConf):
     def __init__(self):
-        if core_supports_xdg():
-            path = MYCROFT_XDG_USER_CONFIG
-        else:
-            path = MYCROFT_USER_CONFIG
-            # mark1 runs as a different user
-            sysconfig = MycroftSystemConfig()
-            platform_str = sysconfig.get("enclosure", {}).get("platform", "")
-            if platform_str == "mycroft_mark_1":
-                path = "/home/mycroft/.mycroft/mycroft.conf"
-
-        if not isfile(path) and isfile(MYCROFT_OLD_USER_CONFIG):
-            # xdg might be disabled in HolmesV compatibility mode
-            # or migration might be in progress
-            # (user action required when updated from a no xdg install)
-            path = MYCROFT_OLD_USER_CONFIG
-
+        path = find_user_config()
         super().__init__(path)
 
 
 class MycroftDefaultConfig(ReadOnlyConfig):
     def __init__(self):
-        mycroft_root = search_mycroft_core_location()
-        if not mycroft_root:
-            raise FileNotFoundError("Couldn't find mycroft core root folder.")
-        path = MYCROFT_DEFAULT_CONFIG.replace("{ROOT_PATH}", mycroft_root)
+        path = find_default_config()
         super().__init__(path)
         if not self.path or not isfile(self.path):
             LOG.error("mycroft root path not found, could not load default "
```

Comment:
Should this create the file path? On first run, I just hit this error on every skill load attempt.
Also, logging the full path here could be useful in debugging

Reply:
it should not create it, but logging it might be a good idea

Reply:
if you are hitting that error then theres some deeper issue...

Reply:
I don't know that neon setup ever creates a user-level `neon.conf`. This is probably a separate issue though if we don't want to create it here

Reply:
this is the default conf that should be shipping with the core its running in, it should always be present

Reply:
Hmm, from what I can tell I'm getting all config from `.venv/lib/.../neon_core/config/neon.conf`.. Keep in mind I am overriding with values from yml config most everywhere in core modules..